### PR TITLE
feat(proxy-cache): the cache_zone field in the schema should be optional

### LINKED ISSUE #2767: DISCUSS(proxy-cache): The cache_zone field in the schema should be optional
### Issue description

The configuration field `cache_zone` of the cache zone in the plug-in is a required item. I think this is unfriendly to some users who are not familiar with this field. They don't know what needs to be done here. In my opinion, `cache_zone` should be optional and have a default value. In this way, for novice users, they do not need to care about what needs to be configured in this field. For familiar users, they can set this field according to their needs.

```
......
properties = {
        cache_zone = {
            type = "string",
            minLength = 1
        },
......
},
required = {"cache_zone"},
```

### PR DESCRIPTION
**Pull request by Firstsawyou**
fix #2767

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: apisix/plugins/proxy-cache.lua [hunk 1]
```diff
@@ -34,7 +34,8 @@ local schema = {
     properties = {
         cache_zone = {
             type = "string",
-            minLength = 1
+            minLength = 1,
```
Comment (nic-chen) [inline apisix/plugins/proxy-cache.lua:37]: should set a max length too.

File: t/plugin/proxy-cache.t [hunk 2]
```diff
@@ -94,9 +120,9 @@ __DATA__
        }
 --- request
 GET /t
---- error_code: 400
---- response_body eval
-qr/failed to check the configuration of plugin proxy-cache/
+--- error_code: 200
```
Comment (nic-chen) [inline t/plugin/proxy-cache.t:123]: we could remove this line when it's 200

File: apisix/plugins/proxy-cache.lua (hunk 1 above)
Comment (Firstsawyou) [reply apisix/plugins/proxy-cache.lua:37]: done.

File: t/plugin/proxy-cache.t (hunk 2 above)
Comment (Firstsawyou) [reply t/plugin/proxy-cache.t]: done.

File: doc/zh-cn/plugins/proxy-cache.md [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 | 名称               | 类型           | 必选项 | 默认值                    | 有效值                                                                          | 描述                                                                                                                               |
 | ------------------ | -------------- | ------ | ------------------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
-| cache_zone         | string         | 必须   |                           |                                                                                 | 指定使用哪个缓存区域，不同的缓存区域可以配置不同的路径，在conf/config.yaml文件中可以预定义使用的缓存区域                           |
+| cache_zone         | string         | 可选   |        disk_cache_one     |                                                                                 | 指定使用哪个缓存区域，不同的缓存区域可以配置不同的路径，在conf/config.yaml文件中可以预定义使用的缓存区域                           |
```
Comment (moonming) [inline doc/zh-cn/plugins/proxy-cache.md:34]: Need more details about how to use this config

File: doc/plugins/proxy-cache.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ The proxy-cache plugin, which provides the ability to cache upstream response da
...
-| cache_zone         | string         | required    |                           |                                                                                 | Specify which cache area to use, each cache area can be configured with different paths. In addition, cache areas can be predefined in conf/config.yaml file                                                                                 |
+| cache_zone         | string         | optional    |  disk_cache_one           |                                                                                 | Specify which cache area to use, each cache area can be configured with different paths. In addition, cache areas can be predefined in conf/config.yaml file                                                                                 |
```
Comment (gxthrj) [inline doc/plugins/proxy-cache.md:34]: We need to tell people the cache will be invalid if `cache_zone` do not match.

**Review by gxthrj — APPROVED** [review]

File: doc/zh-cn/plugins/proxy-cache.md (hunk 3 above)
Comment (Firstsawyou) [reply doc/zh-cn/plugins/proxy-cache.md:34]: I will update later.

File: doc/plugins/proxy-cache.md (hunk 4 above)
Comment (Firstsawyou) [reply doc/plugins/proxy-cache.md:34]: Nice, this sounds great.

**Review by membphis — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**